python/rpdk/java/templates/init/guided_aws/StubUpdateHandler.java

```diff
@@ -0,0 +1,139 @@
+package {{ package_name }};
+
+// TODO: replace all usage of SdkClient with your service client type, e.g; YourServiceAsyncClient
+// import software.amazon.awssdk.services.yourservice.YourServiceAsyncClient;
+
+import software.amazon.awssdk.awscore.AwsRequest;
+import software.amazon.awssdk.awscore.AwsResponse;
+import software.amazon.awssdk.awscore.exception.AwsServiceException;
+import software.amazon.awssdk.core.SdkClient;
+import software.amazon.cloudformation.exceptions.CfnGeneralServiceException;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ProxyClient;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+
+public class UpdateHandler extends BaseHandlerStd {
+    private Logger logger;
+
+    protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
+        final AmazonWebServicesClientProxy proxy,
+        final ResourceHandlerRequest<ResourceModel> request,
+        final CallbackContext callbackContext,
+        final ProxyClient<SdkClient> proxyClient,
+        final Logger logger) {
+
+        this.logger = logger;
+
+        final ResourceModel model = request.getDesiredResourceState();
+
+        // TODO: Adjust Progress Chain according to your implementation
+
+        // STEP 1 [initialize a proxy context]
+        return proxy.initiate("{{ call_graph }}::{{ operation }}", proxyClient, model, callbackContext)
+
+            // STEP 2 [TODO: construct a body of a request]
+            .translate(Translator::translateToUpdateRequest)
+
+            // STEP 3 [TODO: make an api call]
+            .call(this::updateResource)
```

Comment:
I'm still confused why this first update has the call chain already set up but the second update doesn't have a call chain.  I just think it should be consistent.  So maybe instead:
```
ProgressEvent.progress(model, callbackContext)
.then(progress -> firstUpdate(proxy, request, progress))
.then(progress -> secondUpdate(proxy, request, progress))
.then(progress -> new ReadHandler().handleRequest(proxy, request, callbackContext, proxyClient, logger));
```

Then, inside the firstUpdate and secondUpdate methods, you can already have the call chain setup with proxy.initiate.  Same for all handlers, including create/postCreate and delete

Reply:
I dont think we should have different approaches for different handlers. The second update might not be necessary, so it's optional. We are just showing the way to use it, then a developer can modify as preferred.

Reply:
I'm not suggesting different approaches for different handlers, i'm suggesting that we **don't have two different approaches in the same handler**.  First, you call out to a method which doesn't need a call chain, and then you call out to a method which DOES need a call chain (and then we link to an example which doesn't use the call chain)